src/pattern_matching/pssm/mod.rs

```diff
@@ -0,0 +1,154 @@
+// Copyright 2018 Kieran Hervold
+// Licensed under the MIT license (http://opensource.org/licenses/MIT)
+// This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! Implementation of position-specific scoring matrix (PSSM), aka position
+//! weight matrix (PWM), for both DNA and amino-acid sequences
```

Comment:
Could you add an example usage here?

Reply:
How do those examples and doc strings look?
I also cleaned up error-handling a bit -- eg, an erroneous base (eg, an N) used to cause a panic!, but now just returns Err